src/System.Memory/src/System/Span.cs

```diff
@@ -424,8 +424,8 @@ public T[] ToArray()
         //   _pinnable   = null
         //   _byteOffset = the pointer
         //
-        private readonly Pinnable<T> _pinnable;
-        private readonly IntPtr _byteOffset;
-        private readonly int _length;
+        internal readonly Pinnable<T> _pinnable;
+        internal readonly IntPtr _byteOffset;
+        internal readonly int _length;
```

Comment:
Instead of making these internal, could we add the operator to Span<T> and use ROS's dangerous create to create ROS?